starboard/linux/x64x11/gcc/6.3/starboard_loader.cc

```diff
@@ -13,8 +13,20 @@
 // limitations under the License.
 
 #include <stdio.h>
+#include <iostream>
+
 #include "starboard/event.h"
 
+// Provide a workaround alias for newer libcxx that places iostreams in
+// a namespace.
+namespace std {
+// This refers to a macro that should normally resolve to __1, __2 or be
+// defined by LIBCXX_ABI_NAMESPACE or LIBCXX_ABI_VERSION
+namespace _LIBCPP_ABI_NAMESPACE {
+__attribute__((visibility("default"))) std::istream& cin = ::std::cin;
+}
+}  // namespace std
+
 int main(int argc, char** argv) {
   printf("starboard_loader: SbRunStarboardMain\n");
```

Comment:
Not your change ofc, but this seems like an unnecessary printf that can be removed

Reply:
Cleaned that up.